lib/coins/vtc.js

```diff
@@ -0,0 +1,89 @@
+/*
+  info from:
+    https://github.com/vertcoin/vertcoin/blob/master/src/chainparams.cpp
+*/
+
+var common = {
+  name: 'Vertcoin',
+  unit: 'VTC'
+}
+
+var main = Object.assign({}, {
+  hashGenesisBlock: '4d96a915f49d40b1e5c2844d1ee2dccb90013a990ccea12c492d22110489f0c4',
+  // nDefaultPort
+  port: 5889,
```

Comment:
I was not sure about `portRpc`, so I left it out. I could not find in the Vertcoin C++ where it was defined, and I didn't think we needed it at all.